stdlib/buffer.gr

```diff
@@ -300,8 +303,8 @@ export let rec addChar = (char: Char, buffer: Buffer) => {
       let (*) = WasmI32.mul
       let (&) = WasmI32.and
       let (>>) = WasmI32.shrU
-      for (let mut i = 0n; i < 3n; i = i + 1n) {
-        let c = Conv.toInt32(n >> (i * 8n) & 0xffn)
+      for (let mut i = 0n; i < 3n; i += 1n) {
+        let c = Conv.toInt32(n >> i * 8n & 0xffn)
```

Comment:
Hah, this totally gives me second thoughts about the parens since even I don't remember the precedence here 😂